app/src/main/java/org/sopt/melon/presentation/mixup/component/MixUpPlayingMusic.kt

```diff
@@ -0,0 +1,161 @@
+package org.sopt.melon.presentation.mixup.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.melon.R
+import org.sopt.melon.core.common.util.noRippleClickable
+import org.sopt.melon.core.designsystem.theme.MELONTheme
+import org.sopt.melon.presentation.mixup.data.MusicInfo
+
+@Composable
+fun MixUpPlayingMusic(
+    musicInfo: MusicInfo,
+    onAddAllClick: () -> Unit,
+    isAllSelected: Boolean,
+    onAllSelectClick: () -> Unit,
+    isCurrentMusicSelected: Boolean,
+    onSelectClick: () -> Unit,
+    onMenuClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+    ) {
+        // top
+        MixUpPlayingMusicTopBar(
+            onAddAllClick = onAddAllClick,
+            modifier = Modifier.fillMaxWidth(),
+        )
+
+        Spacer(Modifier.size(16.dp))
+
+        // select all
+        Row {
```

Comment:
p1 : 여기 이렇게 내부 패딩값이랑 정렬을 맞춰주세요 ~!! 
```suggestion
           Row(
            verticalAlignment = Alignment.CenterVertically,
            horizontalArrangement = Arrangement.spacedBy(4.dp),
        ) {
```